beetsplug/lyrics.py

```diff
@@ -77,6 +78,11 @@ class HTMLParseError(Exception):
 
 # Utilities.
 
+def unichar(i):
+    try:
+        return six.unichr(i)
+    except ValueError:
+        return struct.pack('i', i).decode('utf-32')
```

Comment:
That sounds sensible—people could have invalid entities. But I guess I'm wondering whether UTF-32 is really the right thing to assume—perhaps we should just drop the character? (Eventually, we'll run into a situation where an entity is so crazy it can't even be decoded with UTF-32 packed as a multi-byte integer. :cry:)